setup.py

```diff
@@ -47,7 +47,7 @@ def check_dependencies():
                             'clean'))):
```

Comment:
Sorry, I think I accidentally requested a review when I was browsing this discussion! I actually haven't been involved in this project.